Automattic/wp-calypso
# Performance Profiler: Add screenshot & timeline on report

Related to https://github.com/Automattic/dotcom-forge/issues/8560 and https://github.com/Automattic/dotcom-forge/issues/8691

## Proposed Changes

* add the website screenshot on the top of the page
* add a timeline of screenshots as the page loads below the CWV metrics

## Testing Instructions



* Go to `/speed-test-tool?url=example.com`
* Check that the website thumbnail is shown according to Figma
* Check that the screenshot timeline is show according to Figma
* Note that we only receive a limited number of screenshots from the backend, so we won't be able to render screenshots more granularly

| Tab | Preview |
|--------|--------|
| Desktop | ![CleanShot 2024-08-15 at 16 49 15@2x](https://github.com/user-attachments/assets/6a6677be-b295-4a3f-b131-1714e21c949c) |
| Mobile | ![CleanShot 2024-08-15 at 16 49 08@2x](https://github.com/user-attachments/assets/fe399b95-c370-452b-92cc-2be6a4c7b01a) |

## Pre-merge Checklist
